src/Sulu/Bundle/ActivityBundle/Resources/config/doctrine/Activity.orm.xml

```diff
@@ -26,5 +26,11 @@
         <field name="resourceSecurityContext" column="resourceSecurityContext" type="string" nullable="true" length="191"/>
         <field name="resourceSecurityObjectType" column="resourceSecurityObjectType" type="string" nullable="true" length="191"/>
         <field name="resourceSecurityObjectId" column="resourceSecurityObjectId" type="string" nullable="true" length="191"/>
+
+        <indexes>
+            <index name="timestamp_idx" columns="timestamp"/>
+            <index name="resourceKey_idx" columns="resourceKey"/>
+            <index name="resourceId_idx" columns="resourceId"/>
```

Comment:
Looks like the `resourceLocale` should also have an index: https://github.com/sulu/sulu/blob/e8a62f408dd15e1b89b9bfff63dda2d89cc3597e/src/Sulu/Bundle/ActivityBundle/UserInterface/Controller/ActivityController.php#L152

Reply:
`resourceSecurityContext`, `resourceSecurityObjectType`, `resourceSecurityObjectId` are also be filtered and should I think then have an index.

Reply:
@alexander-schranz fixed